Bento/Views/BentoReusableView.swift

```diff
@@ -1,10 +1,20 @@
+/// Existential type for reusable views (cell, header, footer) in `UITableView`, `UICollectionView`, etc.
 protocol BentoReusableView: AnyObject {
-    var containedView: UIView? { get set }
+    /// The main view to which you add your reusable view’s custom content.
+    /// - Note: This is normally system-provided as `self`'s subview.
     var contentView: UIView { get }
+
+    /// `component`-generated view that will be added to `contentView`.
+    /// - Note: Subtype of this instance can conform to `ViewLifecycleAware` to allow display handling.
+    var containedView: UIView? { get set }
+
+    /// A renderer which generates `containedView`.
```

Comment:
An instance of component does not generate a view. They only bind with one.

Reply:
`AnyRenderable` owns `viewType` which generates `containedView` _indirectly_ .

Reply:
`AnyRenderable` does not "own" `viewType`. We do query the runtime type through an instance property, but it doesn't mean a `Renderable` _instance_ generating the view is semantically right. It is partly because Swift does not support [opening existentials](https://github.com/apple/swift/blob/master/docs/GenericsManifesto.md#opening-existentials) yet, and therefore we need it to be done this way for querying the associated types.

Reply:
```swift
public struct AnyRenderable: Renderable {
    /// The runtime view type of the wrapped `Renderable`.
    public var viewType: NativeView.Type {
        return base.viewType
    }
```

It owns `base` though. (I don't care about owning directly or indirectly)
And `AnyRenderable.viewType.generate()` eventually means "A renderer which generates `containedView`".